app/views/includes/ptaHeader.scala.html

```diff
@@ -22,16 +22,17 @@
   backLinkGaEventAction: Option[String] = None,
   mainHeadingText: String,
   preHeadingText: String,
-  preHeadingAccessibleAnnouncement: String = "")(implicit m: Messages)
+  preHeadingAccessibleAnnouncement: String = "",
+  backLinkContent: Option[Html] = None
```

Comment:
Might be good to name the new parameter something like `customBackLinkContent` - to clarify that it doesn't need to be populated in the general case where a standard back link is sufficient.